source/simulator/newton.cc

```diff
@@ -246,6 +246,12 @@ namespace aspect
                              "$0\\le \\alpha \\le 1$.)  On the other hand, we also want to stay away from $1+\\alpha x=0$, and so modify the choice of "
                              "$\\alpha$ to be $1$ if $x\\ge -c$, and $\\alpha=-\\frac cx$ with a $c$ between zero and one. This way, if $c<1$, we are "
                              "assured that $1-\\alpha x>c$, i.e., bounded away from zero.");
+
+          prm.declare_entry ("Use the Eisenstat Walker method for Picard", "false",
```

Comment:
same here, and you can skip the 'the'. So: "Use Eisenstat Walker method for Picard iterations"